### PR TITLE
New functions: ft_array_pop, ft_array_indexof, ft_strislanum, ft_strisnumeric

### PR DESCRIPTION
**Pull request by jgigault**

### CODE REVIEW

**Comment by jgigault** [comment]
@anouvel @Tix6 @gabkk @abombard 
Ok for merging?

File: srcs/ft_arr_/ft_array_pop.c [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#include "libft.h"
+
+/*
+** This functions may pop `len` arguments from `argv` starting
+** at element number `start`. It returns the number of argument
+** it has succeeded to pop.
+*/
+
+size_t	ft_array_pop(char ***argv, size_t start, size_t len)
```
Comment (gabkk) [inline srcs/ft_arr_/ft_array_pop.c]: size_t nbr or total instead of size_t len will be more explicit

Comment (jgigault) [inline srcs/ft_arr_/ft_array_pop.c]: ok I agree, and i should rename `argv` with `array` to become more generic

File: srcs/ft_arr_/ft_array_indexof.c [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+** the string `*str` with the function `ft_strcmp`.
+** It returns `1` on success and `0` on missing element.
+*/
+
+int	ft_array_indexof(char const **array, char const *str)
+{
+	while(*array)
+	{
+		if (ft_strcmp(*array, str) == 0)
+			return (1);
```
Comment (gabkk) [inline srcs/ft_arr_/ft_array_indexof.c]: Add in int i and return his value;

File: srcs/ft_arr_/ft_array_indexof.c [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+		array++;
+	}
+	return (0);
```
Comment (gabkk) [inline srcs/ft_arr_/ft_array_indexof.c]: return (-1);

**Pull request merged**